propagation_ot.go

```diff
@@ -53,7 +58,8 @@ func (p *splitTextPropagator) Inject(
 		splitTextCarrier.Baggage = make(map[string]string, l)
 	}
 	for k, v := range sc.raw.Baggage {
-		splitTextCarrier.Baggage[k] = v
+		fmt.Println("BHS2", prefixBaggage+k, v)
```

Comment:
ah indeed -- thanks.

Will look at the dapperish test -- I was running `make test` and assuming that was all that needed to be done.

```
$ make test
go test -v -cover ./...
=== RUN   TestSpanPropagator
--- PASS: TestSpanPropagator (0.00s)
PASS
coverage: 67.8% of statements
ok  	github.com/opentracing/basictracer-go	0.018s	coverage: 67.8% of statements
?   	github.com/opentracing/basictracer-go/examples	[no test files]
?   	github.com/opentracing/basictracer-go/examples/dapperish	[no test files]
=== RUN   TestInMemoryRecorderSpans
--- PASS: TestInMemoryRecorderSpans (0.00s)
PASS
coverage: 100.0% of statements
ok  	github.com/opentracing/basictracer-go/testutils	0.017s	coverage: 100.0% of statements
```